modules/log6x-oc-explain.adoc

```diff
@@ -49,7 +49,7 @@ For example:
 $ oc explain lokistacks.loki.grafana.com.spec.size
 ----
 
-This will show that `size` should be defined using an integer value.
+This output shows that the `size` field must be set to a string value.
 
 == Default Values
```

Comment:
I would recommend doing this in another PR if it's something that you want to change, but in either case this is out of scope and up to you - I notice that none of these second level headings have IDs on them, which they should have.

If this is something you want to fix in another PR, I would go through this file and make sure every heading has an ID, like this:
```suggestion
[id="default-values_{context}"]
== Default Values
```